src/Compilers/CSharp/Portable/Syntax/DirectiveTriviaSyntax.cs

```diff
@@ -62,7 +64,7 @@ public SyntaxToken DirectiveNameToken
             }
         }
 
-        public DirectiveTriviaSyntax GetNextDirective(Func<DirectiveTriviaSyntax, bool> predicate = null)
+        public DirectiveTriviaSyntax? GetNextDirective(Func<DirectiveTriviaSyntax?, bool>? predicate = null)
```

Comment:
I'm unsure why this would need to handle a null directive here or below. We know every trivia is a `Directive`, so it shouldn't get passed null.

Reply:
Good catch. Thanks

---
In reply to: [426966603](https://github.com/dotnet/roslyn/pull/44266#discussion_r426966603) [](ancestors = 426966603)